Python/Product/PythonTools/VSPackage.resx

```diff
@@ -326,28 +326,34 @@
     <value>A sample for working with the Pyvot library for Excel</value>
   </data>
   <data name="5018" xml:space="preserve">
-    <value>Classifier Project</value>
+    <value>Classifier Project (cookiecutter)</value>
   </data>
   <data name="5019" xml:space="preserve">
     <value>A project for testing scikit-learn classifiers.
 
-This project requires the pandas, numpy, scikit-learn, and matplotlib packages.</value>
+This project requires the pandas, numpy, scikit-learn, and matplotlib packages.
+
+Opens Cookiecutter window where you can create the project from a pre-cooked template availalble on github.</value>
```

Comment:
"pre-cooked" is not a great term for this. Just make it "from a cookiecutter template available online." (Also, available is spelled wrong.)